functions/Get-DbaPrivilege.ps1

```diff
@@ -4,7 +4,7 @@ function Get-DbaPrivilege {
         Gets the users with local privileges on one or more computers.
 
     .DESCRIPTION
-        Gets the users with local privileges 'Lock Pages in Memory', 'Instant File Initialization', 'Logon as Batch', or 'Generate Security Audits' on one or more computers.
+        Gets the users with local privileges 'Lock Pages in Memory', 'Instant File Initialization', 'Logon as Batch', 'Generate Security Audits' or 'Logon as a service' on one or more computers.
```

Comment:
```suggestion
        Gets the users with local privileges 'Lock Pages in Memory', 'Instant File Initialization', 'Logon as Batch', 'Generate Security Audits', or 'Logon as a service' on one or more computers.
```

Push a small change as the suggestion above and it will get Appveyor to rerun the tests to see if it passes.